### ISSUE TITLE
Each swift project build leaves behind 1.5 GB dangling docker image

### ISSUE THREAD

**Issue by makandre** [eclipse/codewind#2389]
**Codewind version:**
**OS:**

**Che version:**
**IDE extension version:**
**IDE version:**
**Kubernetes cluster:**

**Description:**

Every time a Codewind swift project is built, it leaves behind a 1.5 GB dangling docker image (it corresponds to the `-build` container):

```
andrewmak@andrews-mbp ~ % docker images | grep cw-cwswift
cw-cwswift01-337ca7f0-5d91-11ea-89f3-4bd7c6ac1892             latest                              f84559873625        8 minutes ago       421MB
cw-cwswift01-337ca7f0-5d91-11ea-89f3-4bd7c6ac1892-build       latest                              9a69a4701077        11 minutes ago      1.52GB
andrewmak@andrews-mbp ~ % docker images | grep none      
<none>                                                        <none>                              45e910f770d5        14 minutes ago      1.52GB
<none>                                                        <none>                              4a79002da7aa        20 minutes ago      1.52GB
<none>                                                        <none>                              ee0ea0300411        23 minutes ago      1.52GB
<none>                                                        <none>                              8441689fb2c8        30 minutes ago      1.52GB
```

**Steps to reproduce:**

**Workaround:**

**Comment by makandre** [comment]
Note: old image should be deleted *after* new image is built in order to leverage cached docker layers, similar to what's done in https://github.com/eclipse/codewind/pull/2388

**Comment by makandre** [comment]
I put a fix for this into #2388

/assign
